control-plane-operator/controllers/hostedcontrolplane/nto/clusternodetuningoperator.go

```diff
@@ -0,0 +1,209 @@
+package nto
+
+import (
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/manifests"
+	"github.com/openshift/hypershift/support/config"
+	"github.com/openshift/hypershift/support/metrics"
+	"github.com/openshift/hypershift/support/util"
+	prometheusoperatorv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+const (
+	operatorName       = "cluster-node-tuning-operator"
+	metricsServiceName = "node-tuning-operator"
+)
+
+type Images struct {
+	NodeTuningOperator string
+	NodeTunedContainer string
+}
+
+type Params struct {
+	ReleaseVersion          string
+	AvailabilityProberImage string
+	HostedClusterName       string
+	Images                  Images
+	DeploymentConfig        config.DeploymentConfig
+}
+
+func NewParams(hcp *hyperv1.HostedControlPlane, version string, images map[string]string, setDefaultSecurityContext bool) Params {
+	p := Params{
+		Images: Images{
+			NodeTuningOperator: images["cluster-node-tuning-operator"],
```

Comment:
can you move cluster-node-tuning-operator into a const?

Reply:
Thanks, I fixed this.